From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: "Brett T. Warden" <brett.t.warden@intel.com>
Date: Tue, 6 Nov 2018 16:43:22 -0800
Subject: [PATCH] Use system mkldnn

[Update for 2019-08-01]: rebased against 1.5.0 release
[Update for 2020-06-23]: rebased against 1.6.0 release
[Update for 2020-10-19]: rebased against 1.7.0 release

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 CMakeLists.txt | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 437d016..6a19098 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -290,10 +290,6 @@ if(USE_MKLDNN)
       set(MKLDNN_CPU_RUNTIME SEQ CACHE INTERNAL "" FORCE)
   endif()
 
-  add_subdirectory(3rdparty/mkldnn)
-
-  include_directories(3rdparty/mkldnn/include)
-  include_directories(${PROJECT_BINARY_DIR}/3rdparty/mkldnn/include)
   add_definitions(-DMXNET_USE_MKLDNN=1)
   list(APPEND mxnet_LINKER_LIBS dnnl)
   set(INSTALL_MKLDNN ON)
